aries_cloudagent/protocols/present_proof/v2_0/routes.py

```diff
@@ -1234,8 +1236,10 @@ async def present_proof_remove(request: web.BaseRequest):
     pres_ex_record = None
     try:
         async with context.profile.session() as session:
-            pres_ex_record = await V20PresExRecord.retrieve_by_id(session, pres_ex_id)
-            await pres_ex_record.delete_record(session)
+            storage = session.inject(BaseStorage)
+            storage_record = StorageRecord(type=V20PresExRecord.RECORD_TYPE, value=None, id=pres_ex_id)
```

Comment:
I think you refactor this to something like:
`storage_record = await storage.get_record(record_type=V20PresExRecord.RECORD_TYPE, record_id=pres_ex_id)`